prow/cmd/tide/main.go

```diff
@@ -68,6 +71,9 @@ func gatherOptions() options {
 	flag.StringVar(&o.jobConfigPath, "job-config-path", "", "Path to prow job configs.")
 	flag.StringVar(&o.cluster, "cluster", "", "Path to kube.Cluster YAML file. If empty, uses the local cluster.")
 
+	flag.IntVar(&o.syncThrottle, "sync-hourly-tokens", 800, "The number of tokens per hour to be used by the sync controller.")
```

Comment:
minor nit: this is the upper bound right? maybe "The maximum number of tokens per hour .."? between the flag and the description this sort of reads like a target. same for the flag below.

Reply:
these also probably need a note in announcements?

Reply:
> these also probably need a note in announcements?

I don't think so? This change is not breaking and only slightly increases the burst size limit for the sync controller, the hourly limits are unchanged. This shouldn't affect any other users of Prow unless flags are specified.